client/app/components/group-detail/component.js

```diff
@@ -0,0 +1,32 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+    editMode : false,
+    resetModalCache(){
```

Comment:
Same as above -- should this say `resetModelCache` instead of `resetModalCache`?